### PR TITLE
feat(frontend): Add information about Blue Papers #385

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- resolves #385

### Steps to test
1. Go to Aprender EOSIO
2. Go to Blue Papers

#### CheckList
- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Comment by JAMares** [comment]
I have doubts about the label

File: docs/aprender-eosio/fundacion-red-eos.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+id: fundacion-red-eos
+title: Fundacion Red EOS Blue Papers
+sidebar_label: Blue Papers
+---
+
+La fundacion de la Red EOS tambien conocidad por sus siglas en ingles como ENF(EOS Network Foundation) es una fundacion sin fines de lucro, han creado una serie de documentos "Blue Papers" sobre los principales temas o dudas de la comunidad y de las respectivas acciones a tomar ante dichas situaciones. En esta seccion veremos de manera consisa y rapida algunos de ellos:
```
Comment (xavier506) [inline docs/aprender-eosio/fundacion-red-eos.md:7]: inluir un espacio aqui ENF(E

**Review by xavier506 — CHANGES_REQUESTED** [review]
Nice Work! 💯 

please see my comments for minor changes

**Comment by JAMares** [comment]
Minor changes fixed

File: docs/aprender-eosio/fundacion-red-eos.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+id: fundacion-red-eos
+title: Fundacion Red EOS Blue Papers
+sidebar_label: Blue Papers
```
[the remark this answers is not in the document]
Comment (xavier506) [reply docs/aprender-eosio/fundacion-red-eos.md:4]: see #351

**Comment by JAMares** [comment]
Missing meta description and keywords fixed

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**